DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Fields/ContentPickerFieldGraphSyncer.cs

```diff
@@ -196,9 +249,14 @@ private async Task<ContentItem[]> GetLatestContentItemsFromIds(JArray contentIte
         }
 
         private object GetNodeId(ContentItem pickedContentItem, IGraphMergeContext context)
-        {
+        {  
+            context.SyncNameProvider.ContentType = pickedContentItem.ContentType;
+            var syncSettings = context.SyncNameProvider.GraphSyncPartSettings;
+
+            _preExistingContentItemVersion.SetContentApiBaseUrl(syncSettings.PreExistingNodeUriPrefix ?? context.ContentItemVersion.ContentApiBaseUrl);
```

Comment:
`PreExistingContentItemVersion`, should only be used for preexisting content items.
If `PreExistingNodeUriPrefix` is not set, then context.ContentItemVersion should be passed to `GetIdPropertyValue`.